### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vldtsw**

### CODE REVIEW

File: templates/includes/sidebar.html [hunk 1]
```diff
@@ -1,4 +1,13 @@
-<ul class="sidebar-nav">
+<ul class="sidebar-nav" style="list-style-type:none">
+    {% if user.is_authenticated %}
+        <li>User: {{ user.username }}</li>
```
Comment (Nazar-kob) [inline templates/includes/sidebar.html:3]: Create a link to user details here

**Review by Nazar-kob — CHANGES_REQUESTED** [review]

File: templates/includes/sidebar.html (hunk 1 above)
Comment (vldtsw) [reply templates/includes/sidebar.html]: My bad, fixed

**Review by Nazar-kob — APPROVED** [review]
Nice job!

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Review by IvanRamyk — APPROVED** [review]
Good job!

**Pull request closed without merging**